RealmSwift/Tests/SwiftTestObjects.swift

```diff
@@ -442,6 +442,18 @@ class SwiftObjectiveCTypesObject: Object {
     @objc dynamic var numCol: NSNumber? = 0
 }
 
+class SwiftComputedPropertyNotExplicitlyIgnoredObject: Object {
+    dynamic var _urlBacking = ""
+    var url: URL? {
```

Comment:
Sorry, do you mean `url` should be marked as `dynamic`, or that we should have two properties, one marked `dynamic` and one not?